packages/frontend-v2/components/ScheduleCourse/ScheduleCourse.tsx

```diff
@@ -19,6 +26,8 @@ interface DraggableScheduleCourseProps {
 export const DraggableScheduleCourse: React.FC<
   DraggableScheduleCourseProps
 > = ({
+  coReqErr,
+  preReqErr,
```

Comment:
```suggestion
  coReqErr = undefined,
  preReqErr = undefined,
```
Lint fix part 2. Basically a draggable course doesn't always have coReq/preReq errors and one of it's usages in the sidebar was erroring in the lint.